Doc/library/asyncio-eventloop.rst

```diff
@@ -1260,8 +1260,9 @@ Executing code in thread or process pools
 
    Arrange for *func* to be called in the specified executor.
 
-   The *executor* argument should be an :class:`concurrent.futures.Executor`
-   instance. The default executor is used if *executor* is ``None``.
+   The *executor* argument should be a :class:`concurrent.futures.Executor`
+   instance. The default executor :class:`concurrent.futures.ThreadPoolExecutor`
+   is used if not set by :meth:`loop.set_default_executor`.
```

Comment:
Sorry to butt in, but this is still pretty imprecise. First the `executor` argument may also be `None`. Second, the default executor is not just any `ThreadPoolExecutor`, it is the default executor *of the event loop* (`loop`), which is always a `ThreadPoolExecutor`. This may indeed be set using `set_default_executor`, otherwise it is "lazy-initialized" by `run_in_executor` if needed.

Reply:
Thanks for the review, how about this:

```
The *executor* argument should be a :class:`concurrent.futures.Executor`
instance. The default executor of the event loop is used if *executor* is ``None``.
The default executor is :class:`concurrent.futures.ThreadPoolExecutor`
and can be set by :meth:`loop.set_default_executor`.
```